_config.yml

```diff
@@ -93,6 +93,7 @@ mermaid:
   # See also docs/ui-components/code
   # To load mermaid from a local file use the `path` key to specify the location of the library instead; e.g.
   # path: "/assets/js/mermaid.min.js"
+  # note: you must still specify a version! Behaviour is different for versions < 10 and >= 10
```

Comment:
```suggestion
  # Note: copy both `mermaid.min.js` and `mermaid.min.js.map` from the specified version of `mermaid/dist` to `/assets/js/`.
  # (2023-04-09: These files are currently unavailable for some releases of mermaid v10.)
```

Reply:
It should be the ESM version, not the UMD bundle; I'll apply a different version of this suggestion locally.